bot/decorators.py

```diff
@@ -154,8 +157,34 @@ async def inner(self: Cog, ctx: Context, *args, **kwargs) -> None:
             log.trace(f"Redirecting output of {ctx.author}'s command '{ctx.command.name}' to {redirect_channel.name}")
             ctx.channel = redirect_channel
 
-            if ping_user:
-                await ctx.send(f"Here's the output of your command, {ctx.author.mention}")
+            async with ClientSession() as session:
+                try:
+                    paste_response = await send_to_paste_service(
+                        files=[PasteFile(content=ctx.message.content, lexer="markdown")],
+                        http_session=session,
+                    )
+                except PasteUploadError:
+                    log.exception(
+                        "Failed to upload message %d in channel %d to paste service when redirecting output",
+                        ctx.message.channel.id, ctx.message.id
+                    )
```

Comment:
Arguments wrong way around, this was probably my snippet that I wrote on mobile, apologies.

```suggestion
                    log.exception(
                        "Failed to upload message %d in channel %d to paste service when redirecting output",
                        ctx.message.id, ctx.message.channel.id
                    )
```